integration-tests/src/test/resources/features/action-sendtobridge.feature

```diff
@@ -0,0 +1,60 @@
+Feature: SendToBridge Action tests
+
+  Background:
+    Given authenticate against Manager
+    And create a new Bridge "bridge1"
+    And the Bridge "bridge1" is existing with status "ready" within 4 minutes
+    And the Ingress of Bridge "bridge1" is available within 2 minutes
+
+  Scenario: Send Cloud Event to SendToBridge with bridgeId defined
+    Given create a new Bridge "bridge2"
+    And the Bridge "bridge2" is existing with status "ready" within 4 minutes
+    And the Ingress of Bridge "bridge2" is available within 2 minutes
+
+    And add a Processor to the Bridge "bridge1" with body:
+    """
+    {
+      "name": "sendToBridgeWithBridgeIdProcessor",
+      "action": {
+        "parameters": {
+            "bridgeId":  "${bridge.bridge2.id}"
```

Comment:
Could we please mention this in readme? We have there currently just env placeholders and some cloud event IDs.

https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/integration-tests/README.md#feature-file-placeholders

Reply:
added